web/wp-content/themes/dctx2/inc/template-tags.php

```diff
@@ -472,6 +472,50 @@ function dctx_display_numeric_pagination( $args = array() ) {
 	<?php
 }
 
+
+/**
+ * Displays post pagination on single posts.
+ *
+ * @param array $args Array of params to customize output.
+ *
+ * @author JC Palmes
+ */
+function dctx_the_post_navigation( $args = array() ) {
+	$args = wp_parse_args( $args, array(
+		'prev_text'          => '<img src="' . get_bloginfo( 'stylesheet_directory' ) . '/assets/images/arrow-prev.png" class="arrow" /> Older',
+		'next_text'          => 'Newer  <img src="' . get_bloginfo( 'stylesheet_directory' ) . '/assets/images/arrow-next.png" class="arrow" />',
+		'in_same_term'       => false,
+		'excluded_terms'     => '',
+		'taxonomy'           => 'category',
+		'screen_reader_text' => __( 'Post navigation' ),
+	) );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).